react/src/views/BoxCreate/BoxCreateView.tsx

```diff
@@ -65,25 +69,84 @@ export const ALL_PRODUCTS_AND_LOCATIONS_FOR_BASE_QUERY = gql`
 `;
 
 function BoxCreateView() {
+  // Basics
+  const navigate = useNavigate();
+
+  // variables in URL
   const baseId = useParams<{ baseId: string }>().baseId!;
-  const [searchParams] = useSearchParams();
-  const qrCode = searchParams.get("qrCode") as string | undefined;
+  const qrCode = useParams<{ qrCode: string }>().qrCode!;
```

Comment:
Why was it not possible to use the QR code from the URL parameter?

Reply:
It is possible, but with this implementation the QRCode becomes a required instead of an optional parameter.
We can roll this back when we have another use case where we create Boxes without QRCodes.